pyoai (2.4.3-1) unstable; urgency=low

  * source package automatically created by stdeb 0.8.2

 -- Arthur Lutz <arthur.lutz@logilab.fr>  Thu, 10 Nov 2016 15:30:18 +0100
